docs/articles/systemd.md

```diff
@@ -33,17 +33,30 @@ If you want Docker to start at boot, you should also:
 There are a number of ways to configure the daemon flags and environment variables
 for your Docker daemon.
 
-If the `docker.service` file is set to use an `EnvironmentFile`
-(often pointing to `/etc/sysconfig/docker`) then you can modify the
-referenced file.
+The recommended way is to use a systemd drop-in file. These are local files in
+the `/etc/systemd/system/docker.service.d` directory.
```

Comment:
Do you think it's also worthwhile to mention that this is often also just an individual `/etc/systemd/system/docker.service` file? (and that the file works just as well as a drop-in?)